python/py-pymatgen/Portfile

```diff
@@ -21,15 +21,16 @@ long_description \
 
 homepage            https://pymatgen.org/
 
-checksums           rmd160  da02e284efbdd35b8af26d33afbb881849e80682 \
-                    sha256  03d24ebafc21becab376c26de09437dd5cfb7bda7099046194e95b4c9fa35209 \
-                    size    3336765
+checksums           rmd160  1a01f5ce6b92ab2f90f916c397576813e9d5d592 \
+                    sha256  08be542ac5449b956022cf17c1c0488b79d6b885bd1861b9120a2216d4e33548 \
+                    size    2620264
 
-python.versions     37 38 39
+python.versions     38 39 310
 
 if {${name} ne ${subport}} {
     depends_build-append \
         port:py${python.version}-cython \
+        port:py${python.version}-pythran \
```

Comment:
why did you add this - I don't see it anywhere listed by upstream? I guess it only might be because building in trace-mode fails otherwise? That's something I've seen as well and is because `cython` checks whether the package is available (that appears to give the answer "yes" in trace-mode) but then the package in reality cannot be used in trace-mode so it fails. If that's the case, then please remove the dependency here as it is not actually needed but just an artifact of the trace-mode. You can verify this locally by deactivating `pythran` and trying to build again.